src/tools/compiletest/src/runtest.rs

```diff
@@ -723,7 +727,7 @@ impl<'test> TestCx<'test> {
 
     /// Replace line numbers in coverage reports with the placeholder `LL`,
     /// so that the tests are less sensitive to lines being added/removed.
-    fn anonymize_coverage_line_numbers(coverage: &str) -> Cow<'_, str> {
+    fn anonymize_coverage_line_numbers(line: &str) -> String {
```

Comment:
Note that this string is actually the whole coverage output, not just one line.

That's why we use `(?m:^)` to match the start of a line, and then use `Regex::replace_all` to modify all of the matching lines in one call.